airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -266,11 +266,11 @@ def _start_task(self):
         self.arn = response['tasks'][0]['taskArn']
 
     def _try_reattach_task(self):
-        task_def_resp = self.client.describe_task_definition(self.task_definition)
+        task_def_resp = self.client.describe_task_definition(taskDefinition=self.task_definition)
         ecs_task_family = task_def_resp['taskDefinition']['family']
 
         list_tasks_resp = self.client.list_tasks(
-            cluster=self.cluster, launchType=self.launch_type, desiredStatus='RUNNING', family=ecs_task_family
+            cluster=self.cluster, desiredStatus='RUNNING', family=ecs_task_family
```

Comment:
How did you decide which of the two arguments to drop?

Reply:
the op explained it in the PR description. I think it makes sense

Reply:
Got it, must have missed that!